### PR TITLE
Adding project/admins for modh-addon

### PR DESCRIPTION
**Pull request by crobby**
To allow for secret management when running osde2e addon tests

### CODE REVIEW

**Comment by crobby** [comment]
@bbguimaraes @petr-muller ptal, thanks

File: core-services/secrets/osd-modh-addon-secrets/osd-modh-e2e-secrets.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# this is the Namespace in which your Secret will live
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    openshift.io/description: Automation Secrets for MyProject
```
Comment (jeefy) [inline core-services/secrets/osd-modh-addon-secrets/osd-modh-e2e-secrets.yaml:6]: You should change these too :)

Comment (crobby) [reply core-services/secrets/osd-modh-addon-secrets/osd-modh-e2e-secrets.yaml]: Thanks

**Review by petr-muller — APPROVED** [review]

**Pull request merged**